src/aggregate-upgrade.rst

```diff
@@ -1,3 +1,24 @@
+.. spelling::
+
+  NaN
+  dateTime
+  Nan
+  pgAdmin
+  memcache
+  pre
+  init
+  datetime
+  itemset
+  jr
+  ui
+  clickjacking
+  JBoss
+  Resteasy
+  Memcache
+  pre
```

Comment:
Duplicate

Reply:
Yes, I have sorted the ``spelling wordlist``. I missed it here. I will fix that and remove the duplicates as well.